Libraries/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_subdirectory(LibCore)
 add_subdirectory(LibCrypto)
 add_subdirectory(LibDebug)
 add_subdirectory(LibDesktop)
+add_subdirectory(LibEmail)
```

Comment:
`LibIMAP` ;-)

Reply:
@elcuco My intention was to reuse some of the classes I wrote for other email protocols (e.g. SMTP, POP3) in the future, so I'm not sure what to call it. I guess we can always rearrange the library namespaces if in the future more protocols are added. I'll change the name to `LibIMAP` for now to be safe.